src/components/customCharts/StackedBarChart.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useRef } from 'react';
+import {
+  setFontStyle,
+} from './helpers';
+
+interface Props {
+  data: any[];
+  width?: number;
+  height?: number;
+  spaceBetweenBars?: number;
+  barHeight?: number;
+  barColor?: string;
+  selectedBarColor?: string;
+}
+
+const CHART_DETAILS_COLOR = 'rgb(72,72,72)';
+
+const SimpleBarChart = ({
+  data,
+  width = 1200,
+  height = 80,
+  barHeight = 60,
+}: Props) => {
+  const canvasRef = useRef(null);
+  const totalDataValues = data.reduce((acc, next) => acc + next.value, 0);
+  useEffect(() => {
+    if (canvasRef.current) {
+      // @ts-ignore
```

Comment:
To bym jakos w funkcje opakował, bo nie wiadomo co to robi

Reply:
Tak, na pewno będę wydzielał jakieś wspólne i bardziej czytelne funkcje (kilka już jest w 'helpers') ale w poprzednim komentarzu - jeszcze nie zdążyłem i nie chciałem tego robić za szybko, żeby nie zapędzić się w "premature optimisation".